pkg/secrets/secrets.go

```diff
@@ -127,9 +126,33 @@ func getMountsMap(path string) (string, string, error) {
 	return "", "", errors.Errorf("unable to get host and container dir")
 }
 
-// SecretMounts copies the contents of host directory to container directory
+// SecretMounts copies, adds, and mounts the secrets to the container root filesystem
+func SecretMounts(mountLabel, containerWorkingDir string) ([]rspec.Mount, error) {
+	var secretMounts []rspec.Mount
+	// Add secrets from paths given in the mounts.conf files
+	for _, file := range []string{OverrideMountsFile, DefaultMountsFile} {
+		mounts, err := addSecretsFromMountsFile(file, mountLabel, containerWorkingDir)
+		if err != nil {
+			logrus.Warn("error mounting secrets, skipping...")
+		}
+		secretMounts = append(secretMounts, mounts...)
+	}
+
+	// Add FIPS mode secret if /etc/system-fips exists on the host
+	_, err := os.Stat("/etc/system-fips")
+	if err == nil {
+		if err := addFIPSsModeSecret(&secretMounts, containerWorkingDir); err != nil {
+			return nil, errors.Wrapf(err, "error adding FIPS mode secret to container")
+		}
+	} else if os.IsNotExist(err) {
+		logrus.Warn("/etc/system-fips does not exist on host, not mounting FIPS mode secret")
```

Comment:
Drop this to a Debug. I don't think this file exists unless you have FIPS mode enabled, and the vast majority of systems do not have FIPS mode enabled. We shouldn't warn on something this common.

Reply:
fixed